gradle/scripts/lib/java-shade.gradle

```diff
@@ -257,7 +257,7 @@ private Configuration createShadedTestRuntimeConfiguration(
 
     if (recursedProject.tasks.findByName('trimShadedJar')) {
         project.dependencies.add(shadedTestRuntime.name, files(recursedProject.tasks.trimShadedJar.outJarFiles))
-    } else {
+    } else if (recursedProject.tasks.findByName('shadedJar')) {
         project.dependencies.add(shadedTestRuntime.name, files(recursedProject.tasks.shadedJar.archivePath))
```

Comment:
Thanks!

Reply:
I actually wasn't sure if I should edit this - should I edit the other gradle scripts repo instead?

Reply:
Let me send a PR for this to upstream.

Reply:
Fix has been pushed to `master`. Please rebase.